streams/src/test/java/org/apache/kafka/test/ProcessorTopologyTestDriver.java

```diff
@@ -415,8 +413,8 @@ public synchronized long position(TopicPartition partition) {
             }
         };
         // For each store name ...
-        for (String storeName : storeNames) {
-            String topicName = ProcessorStateManager.storeChangelogTopic(applicationId, storeName);
+        for (String storeName : storeToChangelogTopic.keySet()) {
```

Comment:
nit: we can use `entrySet` in the for loop and then use `value()` to get the topic name.